scripts/generate-themes.ts

```diff
@@ -24,8 +24,9 @@ export function getThemeColors(primary: string): ThemeColors {
     '--c-dark-primary-active': dc.darken(0.5).hex(),
     '--c-dark-primary-light': dc.alpha(0.5).hex(),
     '--c-dark-primary-fade': dc.darken(0.1).alpha(0.1).hex(),
-    '--rgb-dark-primary': c.rgb().join(', '),
+    '--rgb-dark-primary': dc.rgb().join(', '),
```

Comment:
was this the bug from Elk?

Reply:
yep that was one part of it; the other was the CSS thing. I made it a separate commit so you can see it more clearly